patterns/1-initial/concept-anchor.md

```diff
@@ -0,0 +1,76 @@
+## Title
+
+Concept Anchor Pattern
```

Comment:
```suggestion
Concept Anchor
```

Reply:
Removing "Pattern" from the title.
Also adding Alias section for the longer title.